packages/utils/src/useMethods.ts

```diff
@@ -229,6 +230,12 @@ export function useMethods<
               case HISTORY_ACTIONS.REDO: {
                 return history.redo(draft);
               }
+              case HISTORY_ACTIONS.CLEAR: {
+                history.clear();
+                return {
+                  ...draft,
+                };
```

Comment:
Need to return a new copy of the state, otherwise collector methods will not be notified.